.gitignore

```diff
@@ -3,6 +3,8 @@
 *.iws         // IntelliJ
 .idea/        // IntelliJ
 .DS_Store     // Mac
+.pbxproj // Xcode Mac
```

Comment:
Are there any other files in .pbxproj we should keep, or should we remove this line?

Reply:
Deleted this line just to be sure, Mac users can clone the repo and build the app.